gulpfile.js

```diff
@@ -260,3 +260,9 @@ gulp.task('e2etest-report', function() {
   }, 5000);
 
 });
+
+gulp.task('build-postbid', function() {
```

Comment:
This is an OAS specific task so let's include "oas" in the task name to avoid any mistaken notion that to run postbid in general you need to do this build task.